os/include/compress_read.h

```diff
@@ -0,0 +1,97 @@
+/****************************************************************************
```

Comment:
Please move this file to os/include/tinyara/binfmt/compression and update in all locations.

Reply:
done